### ISSUE TITLE
Segfault on v doc

### ISSUE THREAD

**Issue by sarjsheff** [vlang/v#11015]
**V version:** 
```
OS: linux, Linux version 5.12.15-arch1-1 (linux@archlinux) (gcc (GCC) 11.1.0, GNU ld (GNU Binutils) 2.36.1) #1 SMP PREEMPT Wed, 07 Jul 2021 23:35:29 +0000
Processor: 4 cpus, 64bit, little endian, Intel(R) Atom(TM) CPU D2550   @ 1.86GHz
CC version: cc (GCC) 11.1.0

getwd: /root/tmp/test
vmodules: /root/.vmodules
vroot: /root/v
vexe: /root/v/v
vexe mtime: 2021-08-01 10:03:34
is vroot writable: true
is vmodules writable: true
V full version: V 0.2.2 77e9ed4

Git version: git version 2.32.0
Git vroot status: weekly.2021.30-72-g77e9ed41-dirty
.git/config present: true
thirdparty/tcc status: thirdparty-linux-amd64 333c14de
```

**What did you do?**

```
mkdir /tmp/vtmp && cd /tmp/vtmp && v init && v doc .
```

**What did you expect to see?**

Generated docs.
 
**What did you see instead?**

```
Change the description of your project in `v.mod`
Complete!
Segmentation fault (core dumped)
```

**Comment by sarjsheff** [comment]
Also i found error in determining the root directory https://github.com/vlang/v/blob/master/vlib/v/doc/module.v#L21 when input_dir is "/".

**Comment by JalonSolov** [comment]
Interesting.  A bug was just fixed recently with `/` - perhaps there's another one in `v doc`.

@spytheman ^^

**Comment by spytheman** [comment]
Fixed in 0ebad47 :
```
0[19:06:08]delian@nemesis: (master) /v/vmaster $ mkdir /tmp/vtmp && cd /tmp/vtmp && v init && v doc .
Change the description of your project in `v.mod`
Complete!
vdoc: No documentation found for .
1[19:07:58]delian@nemesis:  /tmp/vtmp $ tig
```
